frontend/src/views/ClientView.vue

```diff
@@ -1,57 +1,137 @@
 <script setup lang="ts">
 import Button from "primevue/button";
+import Calendar from "primevue/calendar";
 import Card from "primevue/card";
+import Dropdown from "primevue/dropdown";
+import FloatLabel from "primevue/floatlabel";
+import InputNumber from "primevue/inputnumber";
 import InputText from "primevue/inputtext";
+import Textarea from "primevue/textarea";
 import { ref } from "vue";
 
-const clientsList = ref([
-  { id: 1, name: "Artem Kovshov" },
-  { id: 2, name: "Sergej Herbert" },
-  { id: 3, name: "Artem Kovshov" },
-  { id: 4, name: "Daria Eliseeva" },
-  { id: 2, name: "Sergej Herbert" },
-  { id: 3, name: "Artem Kovshov" },
-  { id: 4, name: "Daria Eliseeva" },
-]);
+const selectedGender = ref();
+
+let userInfo = ref({
+  name: null,
+  surname: null,
+  street: null,
+  plz: null,
+  city: null,
+  landline: null,
+  mobil: null,
+  email: null,
+  date: null,
+  text: null,
+});
 
-const value = ref(null);
+const genders = ref([
+  { gender: "Herr" },
+  { gender: "Frau" },
+  { gender: "Frau Dr." },
+  { gender: "Herr dr." },
+]);
 </script>
 
 <template>
-  <div class="flex flex-col overflow-hidden h-full">
-    <div class="card flex w-full gap-x-6 mb-3">
-      <span class="relative grow">
-        <i
-          class="pi pi-search absolute top-2/4 -mt-2 left-3 text-surface-400 dark:text-surface-600"
-        />
-        <InputText v-model="value" placeholder="Kundensuche" class="pl-10 w-full" />
-      </span>
-
-      <Button class="md:hidden" icon="pi pi-user" size="small" rounded aria-label="Kunde" />
-      <Button class="hidden md:inline-block" label="Neuen Kunden erstellen" />
+  <form>
+    <div class="flex flex-row justify-between">
+      <Button label="See contracts" severity="secondary" text raised />
+      <div class="flex gap-x-2">
+        <Button label="Save" text raised />
+        <Button label="Delete" severity="danger" text raised />
+      </div>
     </div>
-    <div class="grow overflow-auto">
-      <Card class="my-2" v-for="client in clientsList" :key="client.id">
+    <div class="grid grid-cols-1 sm:grid-cols-2 sm:gap-2 xl:grid-cols-4 xl:gap-4">
+      <Card class="my-2">
+        <template #content>
+          <div class="mb-4 font-bold">Name</div>
+          <div class="card flex flex-col justify-center gap-y-6">
+            <Dropdown
+              v-model="selectedGender"
+              :options="genders"
+              optionLabel="gender"
+              placeholder="Anrede"
+              class="w-full md:w-[14rem]"
+            />
+            <FloatLabel>
+              <InputText id="username" v-model="userInfo.name" class="w-full" />
+              <label for="username">Vorname</label>
```

Comment:
Can we try using camel-casing in HTML?

Though, maybe first-name makes more sense? user name has a login ring to it 😅 

```suggestion
              <InputText id="user-name" v-model="userInfo.name" class="w-full" />
              <label for="user-name">Vorname</label>
```